docs/source/UserGuide/VUDRM-Admin.md

```diff
@@ -0,0 +1,123 @@
+
+* [DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#drm-policy)
+* [PlayReady DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#playready-drm-policy)
+* [Fairplay DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#fairplay-drm-policy)
+* [Widevine DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#widevine-drm-policy)
+
+### VUDRM Token
+
+In this section, you can create, validate, decrypt, encode and decode VUDRM Tokens.
+
+<img src="../_static/user-guide-images/6vudrmtoken.png" class="image" width="1000" alt="VUALTO Admin - VUDRM Token Page"/>
+
+There are a variety of templates to choose from which will automatically populate the Policy text box, you can then generate a token with those parameters. 
+
+The buttons on the page do the following:
+
+* `Generate`        - This will generate a token to include the policy you have entered
+* `Validate Policy` - This will ensure the policy is valid
+* `Validate Token`  - Validate the token created
+* `Decrypt Token`   - Decrypts the token
+* `Encode Token`    - URL Encodes the token
+* `Decode Token`    - URL Decodes the token
+
+For an in depth guide of how our tokens work - please refer to the [VUDRM Token documentation](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#). 
+
+### VUDRM Encryption Keys
+
+Within this section, you can call the `CPIX Key Provider` to fetch VUDRM Encryption Keys in CPIX XML document format as well as the CPIX keys as JSON which is generated at the same time, both of which are available to copy. 
+
+We also have our `Legacy JSON Key provider` (available on the `Key Provider API` dropdown) to use if your product requires it.
+
+<img src="../_static/user-guide-images/7encryptkeys.png" class="image" width="1000" alt="VUALTO Admin - Encrypt Keys Page"/>
+
+To create an Encryption key, do the following:
+
+* Select which Key Provider API you wish to use by using the `Key Provider API`
+* Select which DRM Providers by changing the relevant providers button to `YES`
+* Enter a relevant name in the `Content ID` text field
+* Click `Generate`
+* Click `Copy` to copy the newly created key
+
+For more information - please refer to our [Encryption Key Provision documentation](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-key-provision.html#).
+
+### VUPLAY Demo
+
+VUPLAY is our internal demo player. By default, a token will be loaded into the player, however, you can copy and paste a token you created in the `VUDRM Token` section of the `Info` page. This can be used to test content with your variation of the token. 
+
+<img src="../_static/user-guide-images/8vuplay.png" class="image" width="1000" alt="VUALTO Admin - VUPLAY Page"/>
+
+The different options are as follows: 
```

Comment:
As they'd never need them I wouldn't bother mentioning the other tabs, just some instructions on how to use the custom stream tab and things  like .mpd being turned in .m3u8 is using safari is all they need.

Reply:
Actioned.